example-gto/code/README.md

```diff
@@ -0,0 +1,33 @@
+# GTO Get Started
+
+This is an auto-generated repository for use in
+[GTO Get Started](https://github.com/iterative/gto/blob/main/README.md).
+It is a step-by-step quick introduction into basic GTO concepts.
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.7+ is required to run code from this repo.
+
+```console
+$ git clone https://github.com/iterative/example-gto
+$ cd example-gto
+```
+
+Now let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with a tool such as
+[virtualenv](https://virtualenv.pypa.io/en/stable/):
+
+```console
+$ virtualenv -p python3 .env
+$ source .env/bin/activate
+$ pip install -r requirements.txt
+```
+
+## To see GTO in action
+
+To understand how this repo is generated, take a look at the
```

Comment:
should we redirect to the Get Started? (high level vs this generate.sh that has a lot of things that are not that relevant)

Reply:
Good idea, added the link.